packages/admin/README.md

```diff
@@ -1,4 +1,4 @@
-# Monorepo version: `@airnode/admin`
+# Monorepo version: `@api3/admin`
 # Stand-alone version: `@api3/airnode-admin`
```

Comment:
What about these monorepo vs standalone version README. 
This will no longer be the case, right?

Reply:
Multiple READMEs are out of date, I'm planning to go through them once all the base concepts are settled.